### DISCUSSION TITLE
Videoformat with alpha channel

### DISCUSSION THREAD

**Discussion by movingwater** [ChurchApps/ChurchAppsSupport#330] [FreeShow] [answered]
Hi, I want to playout videos with alpha channel. ProRes 4444 and hevc (mov) doesn't seem to work. What would be the best format to playout with alpha channel?

**Comment by vassbo** [comment, marked as answer]
Hi, I'm guessing you can use .webm

Support for HEVC should come once electron is updated: https://github.com/ChurchApps/FreeShow/issues/306

**Comment by movingwater** [comment]
Thanks, yes, webm does the trick. locking forward to hevc support.

Another question related to video: Is there a way to don't loop a video? Just pause it on the last frame? If I add media to a slide (not as background media), I don't have any options to change looping. And for background media if looping is deactivated the video hides after it has finished.

**Comment by vassbo** [comment]
You can turn on/off looping for the background in the show "Media" tools on the right. And if you want to to pause at the end and not clear you can turn off "Clear media when finished" in the "Other" settings (in v1.1.8).
